File: tests/fixtures/tree/java_process_handle/input.java
class App {
    void process(Item[] items) {
        if (items.length == 0) {
            log.warn("empty");
            return;
        }
        for (Item item : items) {
            if (item.valid()) {
                handle(item);
            }
        }
    }

    void handle(Item item) {
        if (!item.ready()) {
            return;
        }
        JSON.toString(item);
        Files.write(path, data);
    }
}

File: tests/fixtures/graph/process_handle/input.rs
fn process(items: &[Item]) {
    if items.is_empty() {
        log::warn("empty");
        return;
    }
    for item in items {
        if item.valid() {
            handle(item);
        }
    }
}

fn handle(item: &Item) {
    if !item.ready() {
        return;
    }
    serde_json::to_string(item);
    std::fs::write(path, data);
}
